x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/DownsampleAction.java

```diff
@@ -72,29 +81,33 @@ public class DownsampleAction implements LifecycleAction {
             WAIT_TIMEOUT_FIELD,
             ObjectParser.ValueType.STRING
         );
+        PARSER.declareBoolean(ConstructingObjectParser.optionalConstructorArg(), FORCE_MERGE_INDEX);
     }
 
     private final DateHistogramInterval fixedInterval;
     private final TimeValue waitTimeout;
+    private final boolean forceMergeIndex;
```

Comment:
~Small nit: should we rename this field (and related variables/names) to `shouldForceMergeIndex` (or something similar)? The name `forceMergeIndex` could also be interpreted as the name of the index that will be force-merged.~

I see that this matches the name in `SearchableSnapshotAction`. Consistency wins in this case :+1: